packages/nectar/src/sagas/messages/messages.types.ts

```diff
@@ -4,10 +4,14 @@ export enum MessageType {
   Empty = -1,
   Basic = 1,
 }
-
 export interface SendMessagePayload {
-  peerId: string
   message: ChannelMessage
+  peerId: string
+}
+
+export interface WriteMessagePayload {
+  message: string
+  channelAddress?: string
```

Comment:
Why is this optional?

Reply:
Because if not provided, it gets current channel by default